src/services/ContactRepository.ts

```diff
@@ -127,6 +132,7 @@ class ContactRepository {
         },
       }
     )
+    // When importing a DID do you think we should be creating the document here as well?
```

Comment:
@maudnals Could you please, let me know if I should be creating a DID document here when I am importing? It seems I should be.

Reply:
As we import it, but what does it do?

Reply:
YES! NO! Let's talk, I'm not sure to understand? 🐑

Reply:
Nice talk --> we agree that `ContactRepository` shouldn't be part of this PR. 🦑